system-int-test/src/cmr/system_int_test/utils/generic_util.clj

```diff
@@ -17,31 +16,13 @@
 (defn generic-request
   "This function will make a request to one of the generic URLs using the provided
    provider and native id"
-  ([token provider-id native-id concept-type]
-   (generic-request token provider-id native-id concept-type nil :get))
-  ([token provider-id native-id concept-type document method]
```

Comment:
Why are we removing the token parameter? That is going to break the generic ingest permissions test. It is also going to be impossible to re-write the test because it needs to use various different tokens to test permissions.

Reply:
I have put the token parameter back in.